runtime/doc/message.txt

```diff
@@ -739,6 +739,12 @@ Example: >
 You tried to execute a command that is neither an Ex command nor
 a user-defined command.
 
+							*E905*  >
+  Cannot be changed after nvimrc files are sourced
```

Comment:
Technically it is less accurate, |vimrc| (forgot the link here) refers to step 3 in |startup|, but as plugins have no business changing `encoding`, "after startup" should be clear enough. :+1: